gateway-ha/src/test/java/io/trino/gateway/ha/handler/TestQueryIdCachingProxyHandler.java

```diff
@@ -61,6 +66,63 @@ public void testExtractQueryIdFromUrl()
                 .isNull();
     }
 
+    @Test
+    void testQueryIdFromKill()
+            throws IOException
+    {
+        assertThat(
+                extractQueryIdIfPresent(
+                        prepareMockRequestWithBody("CALL system.runtime.kill_query(query_id => '20200416_160256_03078_6b4yt', message => 'If he dies, he dies')")))
```

Comment:
Can we add another test case which `message` argument contains query_id text pattern?